package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "cordova-plugin-fingerprint-aio",
-  "version": "5.0.1",
+  "version": "5.1.0",
```

```suggestion
  "version": "5.0.1",
```

Please don't change versions in PRs. It's part of the release process.